### PR TITLE
Issue 9

### LINKED ISSUE #9: Eating an Apple

### PR DESCRIPTION
**Pull request by Bramvanosta**
fixes #9

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "watchify": "^3.7.0"
   },
   "scripts": {
-    "bundle-client": "watchify client/assets/js/main.js -o client/assets/js/bundle.js -t rollupify"
+    "bundle-client": "watchify client/assets/js/main.js -o client/assets/js/bundle.js -t rollupify",
+    "eslint": "eslint ./client/assets/js/**/*.js"
```
Comment (DavidBruant) [inline package.json:20]: 👍

File: Snake Cahier des Charges.md [hunk 2]
```diff
@@ -21,6 +21,8 @@ Les joueurs contrôlent une longue et fine ligne semblable à un serpent, qui do
 - Le joueur à la possibilité de rentrer un pseudo (Si le joueur ne rentre rien on lui assigne un pseudo aléatoirement)
 
 ##Map
-- La taille du terrain de jeu est limité dans un rectangle de 1600x900px.
+- La taille du terrain de jeu est limité dans un rectangle de 1500x900px.
 - La couleur de fond sera noir
 - Les pommes en rouge
+
+![Board grid](/docs/images/multisnake_board.png?raw=true "Board grid")
```
Comment (DavidBruant) [inline Snake Cahier des Charges.md:28]: Super nice addition! Thanks for adding this!!

File: client/assets/js/board.js [hunk 3]
```diff
@@ -47,33 +41,36 @@ export default class Board {
...
 	}
 
 	render(board) {
-		setInterval(function(){
+		setInterval(function () {
+            // TEMP: ONLY START MOVING THE FIRST SNAKE FOR TEST PURPOSES
 			board.snakes[0].move();
+            // END TEMP
+
+			board.checkSnakeWithAppleCollision();
```
Comment (DavidBruant) [inline client/assets/js/board.js]: Maybe it'd be better to check for collisions inside the call to `.move`? Not sure. but it feels like moving and checking for collisions should be bundled together

**Comment by DavidBruant** [comment]
I see a commit related to global jQuery because watchify something.
If that's really a problem, I'm fine with removing jquery from the dependencies and adding to the HTML as long as there is a comment explaining why jquery is there.

**Comment by DavidBruant** [comment]
In any case, that's an excellent PR! The image doc is a really great addition!

**Comment by Bramvanosta** [comment]
About the global jQuery thing, this seems to be a problem with watchify and any external (NPM installed package) library. Importing the socket.io library seems to cause the same problem. I will investigate a little bit further, and if I don't find an easy solution, I will add it to the HTML.

File: client/assets/js/board.js (hunk 3 above)
Comment (Bramvanosta) [reply client/assets/js/board.js]: Yes, seems logical to me. I'll modify this.

**Comment by DavidBruant** [comment]
> I will investigate a little bit further, and if I don't find an easy solution, I will add it to the HTML.

Sounds good.

+ conflicts

**Pull request merged**